### ISSUE TITLE
Recovered from

### ISSUE THREAD

**Issue by Mellyelliott1989** [microCOVID/microCOVID#1223]
**Is your feature request related to a situation you're trying to understand with microCOVID? Please describe.**
A clear and concise description of the situation you're trying to understand by using microCOVID.

**Describe the solution you'd like**

**Additional context**

**Relevant Research or Citations**
If you have any data or studies that back this feature, please link them here. (If applicable)

**Comment by Mellyelliott1989** [comment]
This calculator does not make sense to a lot of people without an option to choose if you’ve recovered from
coronavirus.  There are now lots of studies show long lasting immunity and low re-infection rates. Yes, antibodies decrease over time but let’s not forgot T cell immunity. Over 100 studies found here:   https://brownstone.org/articles/79-research-studies-affirm-naturally-acquired-immunity-to-covid-19-documented-linked-and-quoted/

You lose scientific credibility by not including this very important option.

**Comment by sparr** [comment]
Your feature request would likely get more attention if you fill out the template in your top comment.

**Comment by 1000i100** [comment]
This option interest me with theses precisions : 
- recover from how much time ?
- what variant ?

How about if the one i met had it, on year ago... or had it 2 week ago and is now negative ?

**Comment by VvMarciaL** [comment]
agreed.  I'm 7 weeks past finally AG-  (boosted, breakthrough); I was not the only one at a recent gathering in similar situation - being able to specify that would be useful.
